apps/opik-documentation/documentation/docs/cookbook/bedrock.ipynb

```diff
@@ -0,0 +1,307 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating an account on Comet.com\n",
+    "\n",
+    "[Comet](https://www.comet.com/site?from=llm&utm_source=opik&utm_medium=colab&utm_content=openai&utm_campaign=opik) provides a hosted version of the Opik platform, [simply create an account](https://www.comet.com/signup?from=llm&utm_source=opik&utm_medium=colab&utm_content=openai&utm_campaign=opik) and grab you API Key.\n",
+    "\n",
+    "> You can also run the Opik platform locally, see the [installation guide](https://www.comet.com/docs/opik/self-host/overview/?from=llm&utm_source=opik&utm_medium=colab&utm_content=openai&utm_campaign=opik) for more information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install --upgrade opik boto3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import opik\n",
+    "\n",
+    "opik.configure(use_local=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Preparing our environment\n",
+    "\n",
+    "First, we will set up our bedrock client. Uncomment the following lines to pass AWS Credentials manually or [checkout other ways of passing credentials to Boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html). You will also need to request access to the model in the UI before being able to generate text, here we are gonna use the Llama 3.2 model, you can request access to it in [this page for the us-east1](https://us-east-1.console.aws.amazon.com/bedrock/home?region=us-east-1#/providers?model=meta.llama3-2-3b-instruct-v1:0) region."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import boto3\n",
+    "\n",
+    "REGION = \"us-east-1\"\n",
+    "\n",
+    "MODEL_ID = \"us.meta.llama3-2-3b-instruct-v1:0\"\n",
+    "\n",
+    "bedrock = boto3.client(\n",
+    "    service_name=\"bedrock-runtime\",\n",
+    "    region_name=REGION,\n",
+    "    # aws_access_key_id=ACCESS_KEY,\n",
+    "    # aws_secret_access_key=SECRET_KEY,\n",
+    "    # aws_session_token=SESSION_TOKEN,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Logging traces\n",
+    "\n",
+    "In order to log traces to Opik, we need to wrap our Bedrock calls with the `track_bedrock` function:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "from opik.integrations.bedrock import track_bedrock\n",
+    "\n",
+    "os.environ[\"OPIK_PROJECT_NAME\"] = \"bedrock-integration-demo\"\n",
+    "bedrock_client = track_bedrock(bedrock)"
```

Comment:
You can pass the project name to the `track_bedrock` decorator, it's a bit cleaner than using the environment variable